src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -213,8 +213,8 @@ jint ShenandoahHeap::initialize() {
 
   ReservedSpace sh_rs = heap_rs.first_part(max_byte_size);
   if (!_heap_region_special) {
-    os::commit_memory_or_exit(sh_rs.base(), _initial_size, heap_alignment, false,
-                              "Cannot commit heap memory");
+    os::commit_memory_or_exit(sh_rs.base(), _initial_size, heap_alignment, !ExecMem,
+                              "Cannot commit heap memory", mtGC);
```

Comment:
The argument order needs to be changed here and below.

Reply:
Done.